ChainSafe/web3.unity
# Marketplace ux improvement

Changed item population voids to tasks and awaited them for a better UX.
Fixed image issue with minting collection NFTs.

Closes #1101

To Test:
Add the marketplace prefab to a new scene and direct the login scene to it.
Login in with your dashboard email and wallet.
Load collections and marketplaces.

Expected results:
Images and data load fully before the processing menu toggles off.
NFTs can be minted to collections.
